cineast-core/src/main/java/org/vitrivr/cineast/core/data/MediaType.java

```diff
@@ -13,6 +13,7 @@ public enum MediaType {
   IMAGE(1, "i", "image"),
   AUDIO(2, "a", "audio"),
   MODEL3D(3, "m", "3dmodel"),
+  TEXTUREMODEL3D(5, "mt", "3dtexturemodel"),
```

Comment:
Why do we need to distinguish this here? Can't textured and untextured models be handled the same way, depending on the capabilities of the feature and the availability of textures?